external/rootpy

```diff
@@ -1 +1 @@
-Subproject commit f0f123d5ebce3983ef3bd3d5d8c962787ca499b4
+Subproject commit 295bfb2624a3bf286348f61de009d0c0f3dfb323
```

Comment:
i just checked out external/rootpy from upstream/master and did a git diff and the output was the same as this commit... I'm confused.  Is there a simple place to change it back?